uaclient/tests/test_cli_auto_attach.py

```diff
@@ -23,6 +25,17 @@
 M_PATH = "uaclient.cli."
 M_ID_PATH = "uaclient.clouds.identity."
 
+HELP_OUTPUT = textwrap.dedent(
```

Comment:
@mitechie  please review each HELP_OUTPUT block in this PR to make sure this is the messaging we want for each subcommand